legistar/base.py

```diff
@@ -253,6 +253,42 @@ def toTime(self, text) :
         time = pytz.timezone(self.TIMEZONE).localize(time)
         return time
 
+    def search(self, route, item_key, search_conditions):
+        """
+        Base function for searching the Legistar API. You typically
+        will want to specialize this using by setting the route
+        and item key in a wrapper function.
+
+        If you were searching for matters, then this would looke like
+        search('/matters/', 'MatterId', your_search_condition)
```

Comment:
I think the sentence about wrapper functions is a little confusing. May I suggest striking it, in lieu of a format like:

```
"""
Base function for searching the Legistar API.

Arguments:
blah

Examples:
# Search for bills introduced after Jan. 1, 2017
search('/matters/', 'MatterId', "MatterIntroDate gt datetime'2017-01-01'")
"""
```